rotkehlchen/tests/api/test_users.py

```diff
@@ -864,4 +872,4 @@ def test_users_by_name_endpoint_errors(rotkehlchen_api_server, username, db_pass
         contained_in_msg='Not a valid string',
         status_code=HTTPStatus.BAD_REQUEST,
     )
-    assert rotki.user_is_logged_in is False
+    assert rotki.user_is_logged_in is True
```

Comment:
For this section, to test that the errors actually occur a user needs to be logged in.

Otherwise, the error message that will be displayed is `No user is logged in` which is expected because that's what this PR fixes.

In that case, the majority of those test cases are redundant as they won't really check anything because the authenticated decorator is run first.

Reply:
Hmm I see. No the tests are not redundant. They just check that schema validation happens.